relay/nakama/cardinal.go

```diff
@@ -101,6 +115,14 @@ func cardinalCreatePersona(ctx context.Context, nk runtime.NakamaModule, persona
 	tick uint64,
 	err error,
 ) {
+	defer func() {
+		if r := recover(); r != nil {
+			txHash = ""
+			tick = 0
+			err = eris.Errorf("a panic occurred in nakama in the function, cardinalCreatePersona:, %s", r)
+		}
+	}()
+
 	signerAddress := getSignerAddress()
```

Comment:
Nakama will swallow a panic. There was a bug in this function that was just closing the connection. This catches the panic and turns it into an error.

Reply:
where is the panic occurring? do we know whats triggering this?

Reply:
The bug is gone, but this is still here as safety. Actually ALL of our functions should have this as nakama will just hang up the connection. It may get logged in the output though.